calibration/NUTNRN/CGINS-NUTNRN-00366__20250312.csv

```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+366,CC_cal_temp,20.00,source_file 3305-00327-00255-A_SN_0366_Pre-Deployment_NUTNR-N.zip > SNA0366C.CAL; reference update conducted in Clark South Cold Room at 12C
```

Comment:
the pre-deploys above have the prefix NTR- before SN in 1st column, e.g., NTR-1103

Reply:
the glider/auv ones do not have NTR as part of SN